pallets/multisig/src/lib.rs

```diff
@@ -203,6 +210,28 @@ decl_module! {
 
         fn deposit_event() = default;
 
+        fn on_runtime_upgrade() -> Weight {
+            use sp_version::RuntimeVersion;
+
+            fn kill(item: &[u8]) {
+                use frame_support::{StorageHasher, Twox128};
+                let mut prefix = Vec::new();
+                prefix.extend_from_slice(&Twox128::hash(b"multisig"));
+                prefix.extend_from_slice(&Twox128::hash(item));
+                kill_prefix(&prefix)
```

Comment:
This logic exists as `storage_value_final_key` in `pallets/pips/src/lib.rs` -- let's move that function to a common place and use it here (it's also written in a more efficient way there).